integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRetryImprovementMisc.java

```diff
@@ -0,0 +1,471 @@
+        clusterName, DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "True", DOMAIN_VERSION);
+    // check domain complete should only come back to true when the cluster is fully up
+    checkDomainStatusConditionTypeHasExpectedStatus(shortRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "True", DOMAIN_VERSION);
+  }
+
+  /**
+   * Delete one of the cluster's pods. Cluster Available should stay true throughout, because the cluster remains
+   * 'ready enough', but cluster Complete should go false.  Then, once pod auto-restarts and becomes ready,
+   * cluster Complete should become true.  Domain Available and Complete should act the same as the Cluster conditions
+   */
+  @Order(7)
+  @Test
+  @DisplayName("delete one cluster pod and verify the domain and cluster condition")
+  void testDeleteOneClusterPodWithReplicasTo2() {
+    // delete one server pod
+    String podName = managedServerPrefix + 1;
+    assertDoesNotThrow(() -> deletePod(podName, domainNamespace),
+        String.format("delete pod %s in namespace %s failed", podName, domainNamespace));
+
+    // check the cluster and domain Complete should go false
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(shortRetryPolicy, domainUid, domainNamespace,
+        clusterName, DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "False", DOMAIN_VERSION);
+    checkDomainStatusConditionTypeHasExpectedStatus(shortRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "False", DOMAIN_VERSION);
+
+    // check the domain and cluster Available should stay true
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(shortRetryPolicy, domainUid, domainNamespace,
+        clusterName, DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "True", DOMAIN_VERSION);
+    checkDomainStatusConditionTypeHasExpectedStatus(shortRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "TRUE", DOMAIN_VERSION);
+
+    // Wait for the pod restarted
+    checkPodReady(podName, domainUid, domainNamespace);
+
+    // check the cluster and domain complete should become true
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        clusterName, DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "True", DOMAIN_VERSION);
+    checkDomainStatusConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "True", DOMAIN_VERSION);
+  }
+
+  /**
+   * Delete the Admin server pod. Expect domain status to flip to Complete false and Available false,
+   * but Cluster status to remain unchanged throughout, staying at Complete and Available.
+   * Once Admin server pod restarts, the Domain status should become true.
+   */
+  @Order(8)
+  @Test
+  @DisplayName("Delete the admin server pod. Verify the domain and cluster status")
```

Comment:
@DisplayName("Delete the admin server pod. Verify the domain and cluster status") --> DisplayName("Verify the domain and cluster status" in the absence of Admin Server)